who-we-are/index.md

```diff
@@ -0,0 +1,74 @@
+---
+toc: false
+layout: default
+redirect_from:
+    - team/index.md
+---
+
+# Meet us
+
+The Foundation for Public Code was founded by Ben Cerveny, Boris van Hoytema and Arnout Schuijff in 2019 to guide international collaboration among public organizations around codebase design, quality and governance.
+
+Leveraging 30 combined years of experience working with cities and technology-enabled collaboration, Boris and Ben recognized policy and whitepapers were ineffective catalysts in this space. Their landscape analysis revealed that stewardship was the key to converting the massive public investment in open source and collaboration into long-term, sustainable implementation.
+
+Based on their experience of technological innovation in a traditional industry, Arnout and Pieter van der Does (co-founders of the Adyen payments platform) agreed with Ben and Boris that public organizations deserved better software. They joined the Board of Directors to share their expertise and connections.
```

Comment:
I know we introduced Arnout earlier in the text, but reading it quickly, this sounds like the van der Does brothers.